### PR TITLE
HSEARCH-3976 + HSEARCH-3977 Documentation and bugfix for scrolling

### PR DESCRIPTION
**Pull request by yrodiere**
* [HSEARCH-3976](https://hibernate.atlassian.net/browse/HSEARCH-3976): Document scrolling
* [HSEARCH-3977](https://hibernate.atlassian.net/browse/HSEARCH-3977): Scroll always targets all indexes

### CODE REVIEW

File: integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/SearchMultiIndexIT.java [hunk 1]
```diff
@@ -21,6 +24,7 @@
 import org.hibernate.search.util.impl.integrationtest.mapper.stub.BulkIndexer;
 import org.hibernate.search.util.impl.integrationtest.mapper.stub.SimpleMappedIndex;
 import org.hibernate.search.util.impl.integrationtest.mapper.stub.StubMappingScope;
+import org.hibernate.search.util.impl.test.annotation.TestForIssue;
 
 import org.junit.Before;
 import org.junit.Rule;
```
Comment (fax4ever) [inline integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/SearchMultiIndexIT.java:30]: there is an unused import here

Comment (yrodiere) [reply integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/SearchMultiIndexIT.java:30]: Fixed, thanks

**Comment by fax4ever** [comment]
Nice job, +1 to merge from me

**Comment by yrodiere** [comment]
Merged, thanks!

**Pull request merged**